exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -163,45 +180,36 @@ impl JavaServiceRuntime {
     /// Adds required EJB-related arguments to JVM configuration
     fn add_required_arguments(
         args_builder: InitArgsBuilder,
-        ejb_config: EjbConfig,
+        runtime_config: &RuntimeConfig,
     ) -> InitArgsBuilder {
         args_builder
-            .option(&format!("-Djava.class.path={}", ejb_config.class_path))
-            .option(&format!("-Djava.library.path={}", ejb_config.lib_path))
+            .option(&format!(
+                "-Djava.library.path={}",
+                runtime_config.system_lib_path
```

Comment:
If you're passing  empty `java.library.path` - JVM will complain. So I'm using `Option` here (consult code in https://github.com/vitvakatu/exonum-java-binding/tree/ejb-app-configuration )

Reply:
Do we ever need to avoid passing libpath (Option#None)?

Reply:
In native ITs libpath is not needed, because the libraries will be loaded from RUNPATH. Though it might change with packaging.

Reply:
From the adjacent thread,

> It might be provided to the system linker, but JVM still requires java.library.path, doesn't it?

Doesn't it?

Reply:
JNI spec doesn't mention that anywhere but in an example in Chapter 5, the `System` documentation just says that it is "List of paths to search when loading libraries" https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/System.html#getProperties() 🤷‍♂️